portals/application/configs/klear/FaxesList.yaml

```diff
@@ -137,6 +138,15 @@ production:
       class: ui-silk-eye
       disableSave: true
 
+    faxesOutView_screen:
+      <<: *faxesInOutEdit_screenLink
+      parentOptionCustomizer:
+        - IvozProvider_Klear_Options_OptionsCustomizerFilterForFax
+      title: _("View Incoming %s %s", ngettext('Faxfile', 'Faxefiles', 0), "[format| (%item%)]")
```

Comment:
I miss a translation of this string, or it was already translated?

If you rebase post 75a563f76de6aa46e7dc6ca213eac56f859ad8e8 translation files should be clean! :)